src/core/lib/breakpoint.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Breakpoints ordered from smallest to largest
  */
-const breakpoints = ['mobile', 'phablet', 'tablet', 'desktop'] as const;
+const breakpoints = ['mobile', 'phablet', 'tablet', 'desktop', 'wide'] as const;
```

Comment:
the wide breakpoint is used by [pageskins](https://github.com/guardian/frontend/blob/main/common/app/views/fragments/commercial/pageSkin.scala.html#L6)